ansible/roles/letsencrypt/tasks/main.yml

```diff
@@ -96,6 +95,8 @@
   - name: Setup cron job to auto renew
     cron:
       name: Auto-renew SSL
-      job: sudo "{{certbot_dir}}/certbot-auto renew --quiet --no-self-upgrade"
-      hour: 4
+      job: "{{certbot_dir}}/certbot-auto renew --quiet --no-self-upgrade"
```

Comment:
yeah its a little hard for cron because if it succeeds it will still send you email or whatever.